src/main/java/software/amazon/smithy/lsp/SmithyTextDocumentService.java

```diff
@@ -101,14 +128,33 @@ private CompletionItem create(String s, CompletionItemKind kind) {
     return ci;
   }
 
+  public Boolean isExternalJarURI(String uri) {
```

Comment:
[ditto](https://github.com/disneystreaming/smithy-language-server/pull/1/files#r752155564) : if your JarFile-based function works, we could remove the classloader-based one altogether, couldn't we ?